.buildkite/pipelines/pull_request/base_merged_phases.yml

```
@@ -0,0 +1,143 @@
+steps:
+  - command: .buildkite/scripts/steps/build_kibana.sh
+    label: Build Kibana Distribution
+    agents:
+      machineType: c3d-standard-16-lssd
+      zones: us-central1-a,us-central1-b,us-central1-c
```

```suggestion
      zones: us-central1-a,us-central1-b,us-central1-c,us-central1-f
```
At the time when we rolled out the c4 instances, there weren't any available in us-central1-f, which I believe this is modeled from.

It also looks like c4 instances are available in us-central1-f now but that can be updated out of band if needed.

---

I'd say let's roll this out, and test, in the meantime we can also test if `us-central1-f` has it, then we can update it if/when we make this as the default approach